wagtail_localize/operations.py

```diff
@@ -73,7 +73,11 @@ def create_translations(self, instance, include_related_objects=True):
             self.mappings[source].append(translation)
 
             # Determine whether or not to publish the translation.
-            publish = getattr(instance, "live", True)
+            publish_on_translate_setting = getattr(settings, "WAGTAIL_LOCALIZE_PUBLISH_TRANSLATIONS_OF_LIVE_PAGES", True)
```

Comment:
best to keep the setting prefix as `WAGTAILLOCALIZE_` (related: #494)

`WAGTAIL_LOCALIZE_PUBLISH_TRANSLATIONS_OF_LIVE_PAGES` is rather verbose. WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE` may be better and reflect what his happening here.


```python
publish_on_translate_setting = getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True)

if publish_on_translate_setting:
    publish = getattr(instance, "live", True)
else:
   publish = False
```

Reply:
Also the new setting needs documenting in https://github.com/wagtail/wagtail-localize/blob/main/docs/how-to/installation.md

Reply:
@zerolab Done that, thanks!